scenes/main_menu/main_menu_level.gd

```diff
@@ -2,20 +2,29 @@
 extends "../scene_base.gd"
 
 func _update_demo_positions() -> void:
-	var count = $Demos.get_child_count()
+	var count = 0#$Demos.get_child_count()
+	var vis_chd := []
+	for _tp in $Demos.get_children():
```

Comment:
Kind of the same with `_tp`, also the `_` prefix is a convention to indicate a private variable, not really applicable for local variables. I would just call this `teleporter`